critiquebrainz/frontend/views/review.py

```diff
@@ -278,7 +278,7 @@ def create(entity_type=None, entity_id=None):
         if not form.errors:
             flash.info(gettext("Please provide some text or a rating for this review."))
         return render_template('review/modify/write.html', form=form, entity_type=entity_type, entity=entity,
-                               spotify_mappings=spotify_mappings, soundcloud_url=soundcloud_url)
+                               entity_id=entity_id, spotify_mappings=spotify_mappings, soundcloud_url=soundcloud_url)
```

Comment:
If we use `entity.id` in the templates, we don't need to pass this into the template context